addons/udes_stock_cron/models/stock_picking.py

```diff
@@ -75,7 +72,8 @@ def _find_unreservable_moves(self, batch_size=1000, return_fast=False):
 
                     if product not in stock_for_products.keys():
                         stock_for_products[product] = Quant.get_available_quantity(
-                            product, locations
+                            product,
+                            picking.picking_type_id.default_location_src_id,
```

Comment:
I think all pickings are grouped by picking_type_id, so practically at the beginning of function can be set a variable to get the location, instead of accessing it for every pick from picking_type_id

At the beginning of method smth like:
`locations = self.picking_type_id.default_location_src_id`

```suggestion
                            locations,
```

Reply:
Amended, please review.